src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(a) {
+    return function(b) {
```

Comment:
GJ)
It would be better t0 rename arguments to make them clear, but logic is good.